processor/src/main/java/org/mapstruct/ap/internal/util/accessor/AbstractAccessor.java

```diff
@@ -32,9 +35,20 @@
 abstract class AbstractAccessor<T extends Element> implements Accessor {
 
     protected final T element;
+    private final TypeMirror builderType;
 
     AbstractAccessor(T element) {
         this.element = element;
+        if (element.getEnclosingElement().getKind() == ElementKind.CLASS) {
```

Comment:
If you are doing any changes feel free to update the PR. We can sync up along the way 😀